docs/install/run-openmetadata.md

```diff
@@ -206,3 +212,21 @@ If you see the above when attempting to install OpenMetadata, this can be due to
 
 
 If you need support please get in touch on Slack: [https://slack.open-metadata.org/](https://slack.open-metadata.org).&#x20;
+
+## Requirements (Windows)
```

Comment:
can you move this at the top of the page. Such that these requirements are satisfied and installed before they can install the openmetadata docker package